auth/api/http/groups/endpoint_test.go

```diff
@@ -32,6 +32,7 @@ const (
 	secret        = "secret"
 	id            = "testID"
 	loginDuration = 30 * time.Minute
+	instanceID    = "1"
```

Comment:
I am still very sceptical about giving incremental IDs to instances. I think it should be UUIDv4. How are K8s and others enumerating instances? What type of identifier are they using?

@ianmuchyri @dborovcanin

Reply:
This is for the tests. In the codes we use `uuid.new().ID()` to create a uuid
@drasko

Reply:
`instanceID := cfg.InstanceID
	if instanceID == "" {
		instanceID, err = uuid.New().ID()
		if err != nil {
			log.Fatalf("Failed to generate instanceID: %s", err)
		}
	}`

Reply:
@drasko I have changed to UUID. In the case of test we will use constants but while running the code we generate UUID.